src/components/radio/Radio.tsx

```diff
@@ -0,0 +1,46 @@
+import { Component, createSignal, Show } from 'solid-js'
+
+import StyledIcon from './StyledIcon'
+import StyledInput from './StyledInput'
+import StyledLabel from './StyledRadio'
+
+export type IRadioProps = {
+  label?: any
+  labelPosition?: 'right' | 'left'
+  checked?: boolean
+  indeterminate?: boolean
```

Comment:
Setting this to true doesn't seem to change the styles. Does the radio need an indeterminate prop/style?

Reply:
Oh, I'm not sure why I added it lol :D. We don't need it so I'll remove it.

Reply:
@zedaan did you end up removing the `indeterminate` prop?